test/net/connect.cpp

```diff
@@ -110,14 +110,20 @@ TEST_CASE("connect_base works with ipv4") {
     });
 }
 
+int checkError (Error err) {
+    if (err == NetworkError() || err == TimeoutError())
+        return 1;
+    return 0;
```

Comment:
Could you more compactly write this as `return err == NetworkError() or err == TimeoutError();`?